shop4All-backend/src/main/java/com/example/shop4All_backend/Configuration/WebSecurityConfiguration.java

```diff
@@ -52,18 +52,12 @@ public SecurityFilterChain configure(HttpSecurity http, CorsConfiguration corsCo
                 .csrf(AbstractHttpConfigurer::disable)
                 // Configure authorization rules for different endpoints
                 .authorizeHttpRequests(auth -> {
-                    auth.requestMatchers("/").permitAll();
-                    auth.requestMatchers("/registerNewSeller").permitAll();
-                    auth.requestMatchers("/authenticate").permitAll();
-                    auth.requestMatchers("/login").permitAll();
+                    auth.requestMatchers("/registerNewSeller","/registerNewBuyer","/authenticate","/login","/").permitAll();
                     auth.anyRequest().authenticated();
                 })
                 .exceptionHandling((exceptions) -> exceptions
                         .authenticationEntryPoint(jwtAuthentificationEntryPoint)
                 )
-                .sessionManagement((session) -> session
-                        .sessionCreationPolicy(SessionCreationPolicy.STATELESS)
-                )
```

Comment:
The sessionManagement could be kept. Did you need to delete it? I meant to delete .formLogin() method in a future PR.